actions/publish-pkgset.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
```

Comment:
It seems like we can hide some problems if not all of the env vars are set, but this script doesn't fail

```suggestion
set -u

```